server/redis.repository.js

```diff
@@ -76,6 +78,20 @@ const dislikeEvent = async (roomId) => {
     return -1;
 }
 
+/**
+ * Deletes room in Redis store by roomId
+ * 
+ * @param {String} roomId - The room ID.
+ * @returns {Integer} 1 - If room is deleted. 
```

Comment:
Since delAsync(roomId) returns the number of keys deleted (which is the number of unique movies in this case), wouldn't  this value be >1?

Reply:
It returns the total number of keys removed. So since this function only removes the key of roomID passed in, it must return 1. Let me know if that makes sense!

Reply:
gotcha, thanks for clarifying!